code/game/objects/items/crab17.dm

```diff
@@ -30,6 +30,11 @@
 			var/datum/bank_account/B = i
 			B.being_dumped = TRUE
 		new /obj/effect/dumpeet_target(targetturf, L)
+
+		to_chat(user, span_notice("You have activated Protocol CRAB-17."))
+		message_admins("[user] has activated Protocol CRAB-17.")
```

Comment:
Sorry quick nit, might want to make this an `ADMIN_FLW(user)` or a `key_name(user)`, I'm not sure what `user` will show up as and if it'll be useful for admins

Reply:
No problem done with 8794d63bad561e2fb3d258c78d47a34b7929539f